http/src/main/resources/swagger/api-docs.yaml

```diff
@@ -2582,7 +2844,7 @@ components:
         - type: object
           properties:
             persistentDisk:
-              $ref: "#/components/schemas/PersistentDiskConfig"
+              $ref: "#/components/schemas/PersistentDiskRequest"
```

Comment:
this isn't correct...`blockSize` in `PersistentDiskRequest` doesn't apply

Reply:
`PersistentDiskRequest` doesn't have a parameter `blockSize` here, I removed it.

It now corresponds to the ADT
```final case class PersistentDiskRequest(name: DiskName,
                                       size: Option[DiskSize],
                                       diskType: Option[DiskType],
                                       labels: LabelMap)```

Reply:
oh cool..I thought it's shared with createDisk API, which I double checked it's not